scripts/dts/extract/compatible.py

```diff
@@ -54,6 +54,17 @@ def extract(self, node_address, prop, def_label):
                 }
                 insert_defs(node_address, load_defs, {})
 
+        # Generate defines of the form:
+        # #define DT_<COMPAT>_<INSTANCE ID> 1
+        instance = reduced[node_address]['instance_id']
+        for k in instance:
+            i = instance[k]
+            compat_instance = 'DT_' + convert_string_to_label(k) + '_' + str(i)
```

Comment:
So do we want to always have _0 if only one instance is (and will ever be) available (eg for RTC)?
Or can we __also__ generate the alternate 'no index' #define when instance number is 0?
(This would spare the rework of sensors drivers)

Reply:
I always want _0.  I'd rather the naming convention not try to be overly smart.  The extra '_0_' seems minor for the cases that only one instance will ever exist.